integration/test_overload_protection.py

```
@@ -0,0 +1,160 @@
+
+        # Release the pausepoint so threads can complete
+        node0.execute_command("FT._DEBUG", "PAUSEPOINT", "RESET", "background_search_completing")
+
+        for t in threads:
+            t.join(timeout=15)
+
+        for c in clients:
+            try:
+                c.close()
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)